gammapy/estimators/energydependence.py

```diff
@@ -0,0 +1,262 @@
+import numpy as np
+from astropy.table import Table
+from gammapy.datasets import Datasets
+from gammapy.modeling import Fit
+from gammapy.modeling.models import FoVBackgroundModel
+from gammapy.modeling.selection import TestStatisticNested
+from gammapy.stats.utils import ts_to_sigma
+from .core import Estimator
+
+
+def weighted_chi2_parameter(results_edep, parameter="sigma"):
```

Comment:
The idea was that this is an additional calculation that could be made. 
One should really utilise the results_edep significance values because these include any correlation between parameters.

Reply:
If there are caveats to keep in mind, it should be clearly mentioned in the doc string and the tutorial, otherwise everyone will definitely use it.